src/Components/Form/Button.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './form-scss/form-component.scss';
+
+const Button = (props) => {
+  // typechecking with propTypes
+  Button.propTypes = {
+    className: PropTypes.string,
+    type: PropTypes.string,
+  };
+
+  // destructring props
+  const { type, className, onClick, name } = props;
+
+  return (
+    <div>
+      <button
+        className={className}
+        type={type}
+        // onClick={onClick}
```

Comment:
make button component reusable, by using on submit we will be able to use this component for forms only